packages/atlas/src/providers/joystream/joystream.hooks.ts

```diff
@@ -254,6 +254,8 @@ export const useFee = <TFnName extends TxMethodName, TArgs extends Parameters<Jo
     () =>
       debouncePromise(async (args?: TArgs) => {
         if (!args || !accountId) {
+          setFullFee(new BN(0))
+          setLoading(false)
```

Comment:
Why was this change needed? I did a quick test with commenting and this changes the fee display behavior:

https://user-images.githubusercontent.com/12646744/188065724-96084c9b-26b4-400b-a143-95dc98cd8269.mp4

Previously the fee would stay the same when signing the tx. Maybe this actually a problem in the CommentInput, I'm not sure, but we should check.

Reply:
Also if I reject the signature, the fee stays 0